.github/workflows/release.yml

```diff
@@ -2,6 +2,7 @@ name: Weekly Release
 
 on:
   workflow_dispatch:
+  push:
```

Comment:
This is just for testing the PR and should be removed before merging, right?

Reply:
Yes